src/components/common.js

```diff
@@ -362,6 +362,8 @@ const BaseSelect = ({ value, newOptions, id, findValue, maxHeight, ...props }) =
  * @param props.id - The HTML ID to give the form element
  */
 export const Select = ({ value, options, id, ...props }) => {
+  Utils.useConsoleAssert(props.id || props['aria-label'] || props['aria-labelledby'], 'In order to be accessible, Select needs a label')
```

Comment:
Nice, this is very clean.

Reply:
Yeah, actually, at some point this should be made consistent in all of the widgets like this. I'm not sure it is...